### ISSUE TITLE
Exception attempting to get latest data: ‘temp’

### ISSUE THREAD

**Issue by s28400** [andrewleech/uhoo-homeassistant#5]
## Version of the custom_component
0.0.6

## Describe the bug

New exception started a few days ago, might be API change on uHoo end or key change. See below:

## Debug log

```text

This error originated from a custom integration.

Logger: custom_components.uhoo
Source: custom_components/uhoo/__init__.py:86
integration: uHoo (documentation, issues)
First occurred: 11:20:25 PM (1 occurrences)
Last logged: 11:20:25 PM

Error: an exception occurred while attempting to get latest data: 'temp'

```

**Comment by srescio** [comment]
hi folks

TL;DR

1. open you uHoo mobile app
2. tap hamburgher menu
3. Appearance
4. Temperature
5. Change it to whatever else you currently have and save
6. Change it back to your previous setting

Issue is in the pyuhoo library trying to access the temperature unit selection from somewhere that is not anymore guaranteed to be present, reselecting the unit makes the API return again said field, without which the unhandled exeption occurs

I forked the library repo and prepared a fix to infer the unit from somewhere else
https://github.com/andrewleech/pyuhoo/pull/1
am preparing also the bump in the homeassistant component itself:
https://github.com/andrewleech/uhoo-homeassistant/pull/6

**Comment by srescio** [comment]
i think this can be closed now
